geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.result.model;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.apache.commons.lang.NotImplementedException;
+
+import org.apache.geode.management.cli.Result;
+import org.apache.geode.management.internal.cli.CommandResponse;
+import org.apache.geode.management.internal.cli.json.GfJsonObject;
+import org.apache.geode.management.internal.cli.result.ResultData;
+
+public interface ResultModel extends Result, CommandResponse.Data, ResultData {
+
+  String getType();
```

Comment:
since ResultModel extends these other interfaces that already have those methods, it doesn't need re-define these methods again in this interface.

Reply:
Alternatively, if the intent is to eventually remove `ResultData`, I'd note that in the Javadoc, keep the declarations here to represent the intent for this interface t subsume the other, and add the `@Override` until the parent interface can be removed.